src/node/desktop/src/main/menu-callback.ts

```diff
@@ -333,7 +381,7 @@ export class MenuCallback extends EventEmitter {
     };
 
     if (checkable) {
-      menuItemOpts.type = 'checkbox';
+      menuItemOpts.type = 'radio';
```

Comment:
We don't want all checkable menu items to be `radio`, only those marked as such in the Commands.cmd.xml. See comment in the issue: #10876

https://github.com/rstudio/rstudio/blob/09be8fa4417fce8b42aea96d9ed9e1f76470864a/src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml#L31-L33

Reply:
It is odd to have `radio` set for everything when that's not what comes from the `Commands.cmd.xml`. It seemed to work though and even exactly to the odd behaviour that Qt has with the Zoom Source/Help/Console panes when zoomed on another pane.

It probably is worth using the `radio` only when specified. GWT will need an update to specify whether the menu item is checkable or a radio. I just can't find an existing menu item that requires checkable, which is why this seems to work for now using `radio`.

Reply:
Gary pointed me to the checkable menu items. The ones that I think should be tested are the Soft Wraps and Rainbow Parentheses in the Code menu. They are set as checkable and can both be checked at the same time.

Reply:
Yea, that's why I kept radio for everything. As the menu is entirely rebuilt, the items can be unchecked if needed, as it seems that is GWT is handling the state for all menu items and passing over to electron. I thought of keeping everything as radio not to spend more time going around to see that part as I'm not used to GWT but I think that could be good to maintain a better code.

Reply:
There are a bunch of menu items in Commands.cmd.xml annotated as checkable vs. radio. Some might make more sense as `radio`, but there are some that are definitely not radios. Even if it seems to be behaving correctly, it is an accesibility problem for screen reader users to have a single checkable menu item marked as radio (and may also have the wrong visual representation).

I can do the bit of plumbing needed to get the metadata on radio vs. checkable where we need it.

Reply:
For this PR, let's keep the menutype as "checkbox". We will track using "radio" (if we do it at all) via #10876.